### PR TITLE
Article clone

### PR DESCRIPTION
**Pull request by lily-coder**
This is a clone of The Next Week Project
We have completed milestone two and added all the webpage content

### CODE REVIEW

**Review by george-shammar — CHANGES_REQUESTED** [review]
### **STATUS: CHANGES REQUIRED** :recycle: :recycle: 

**Good effort on  the progress you've made so far**. :+1:  **Your project looks really good.** :clap: :clap: 

**However, we need to effect some changes in order to get this project approved.** **Please see below**:

**Latest deals** (**mobile version**)

- [ ] Each article should have an image on the left and the title should be on the right on top of the price. Yours currently have the title below the image. See below: While making the change, please align "see all deals" to be on the same level as the "Latest deals". It is slightly off and higher than the title.


image

**Latest deals** (**tablet version**)
- [ ] Each article should have the image on the top, then the title and the price at the bottom. Yours currently have the image at the top and then the title and the price side by side. Please see below:

Original:
image

Yours:
image

[OPTIONAL]

- [ ] **You disabled stylelint on countless occasion in your style sheet.  Try to correct whatever errors is being thrown up. See my inline comment. There are many more instances where stylelint was disbaled that I couldn't comment. Please go through your styleesheet and correct the errors. It's an important part of learning.**

**Please make these changes and submit another CRR once you're ready.**



As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have 2 code reviews left for this project.

If you require further clarification, please reach on slack @george gbenle.

**Happy coding.** :desktop_computer: :hammer_and_pick: :hammer:

File: style.css [hunk 1]
```diff
@@ -340,202 +455,338 @@ i {
...
-/* stylelint-disable */
-.btm-nav a {
-  margin: 0;
-}
-/* stylelint-enable */
+  /* stylelint-disable */
+  .btm-nav a {
+    margin: 0;
+  }
+  /* stylelint-enable */
```
Comment (jcy2704) [inline style.css]: - [ ] You disabled stylelint here. Please remove it and solve the linters. 

**It is a bad practice to disable the linters, try to solve it. If it is something that you can't even find on the internet on how to solve it, ask the full-time students chat in the Slack group. If no one else can solve it, then there you resort to disabling the linters.**

**Review by jcy2704 — CHANGES_REQUESTED** [review]
# CHANGES REQUIRED 👀🤔

## Hey @lily-coder @Anasscodes!! Great work making the changes, your project looks even more incredible! 🙌

> **REMINDER: THIS IS THE SECOND REVIEW YOU STILL HAVE ONE MORE REVIEW SO PLEASE TRY TO FIX ALL CHANGES BEFORE ASKING FOR A REVIEW.**

### However, there are still some things we need to change before your project gets approved. 🧐

**I want to say that your code looks very clean and organized 😯, I really like that.**

<hr>

<h2 align="center">Changes</h2>

image

image

- [ ] 👆👆There seems to be an issue with some of the images in your demo link try to see what is wrong and fix it.

- [ ] The links in the top navbar are in black so they can not be seen. Please change the font color.

<h2 align="center">OPTIONAL</h3>

image

- [ ] In the mobile resolution, in the hard fork section, there is no margin on the sides. Please add some to match the original

### MORE CHANGES BELOW! 👇

<hr>

### Keep the awesome work! 💪 Happy Coding for y'all!! 💻

**If you have any questions you can ask me on my Slack.**

**Best,**

**[💁‍♂️ Steven Jack Chung Yau](https://stevenchung.me)**

[LinkedIn](https://linkedin.com/in/stevenjchung) | [Twitter](https://twitter.com/_yiak) | [GitHub](https://github.com/jcy2704)

**Review by mricanho — CHANGES_REQUESTED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @lily-coder @Anasscodes, good job on this milestone! 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽
 
nice


Now you can merge this branch 🚀 

You can contact me on slack if you have any questions, just search for me as @ Miguel Ricaño.

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

**Pull request merged**